app/src/main/java/org/mozilla/focus/browser/cursor/CursorController.kt

```diff
@@ -54,7 +54,10 @@ class CursorController(
             KeyEvent.ACTION_UP -> viewModel.onDirectionKeyUp(dir)
             else -> Unit
         }
-    }, dispatchTouchEventOnCurrentPosition = { action -> viewModel.onSelectKeyEvent(action) })
+    }, dispatchTouchEventOnCurrentPosition = { event ->
+        viewModel.onSelectKeyEvent(event.action)
+        view.updateCursorPressedState(event)
```

Comment:
nit: There's discrepancy here: one method says what has happened (onSelectKeyEvent) while the other says what it's going to do (updateCursorPressedState).

I'm not sure which is better – probably the latter, to help with code readability. I think we can leave it the way it is for now, but for future reference.